src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -118,6 +118,10 @@ export const LongButtonText: StoryObj<Args> = {
       </Button>
     ),
   },
+  parameters: {
+    // Is flaky on chromatic, disabling for now
```

Comment:
Think we can! My initial thought was to disable the flakiness first and then play with the delay timing to find a good time, but now I feel that since people haven't been complaining about this flakiness (afaik) I'll just set a delay time instead.